pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -122,56 +122,69 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, nil
 	}
 
+	list := &autoscaling.ProvisioningRequestList{}
```

Comment:
Can we adjust naming so it better express what's underneath?

Reply:
Done